Unity3D/Assets/RosSharp/Scripts/PosePublisher.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
+© Federal Univerity of Minas Gerais (Brazil), 2017
+Author: Lucas Coelho Figueiredo (me@lucascoelho.net)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+<http://www.apache.org/licenses/LICENSE-2.0>.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using UnityEngine;
+
+namespace RosSharp.RosBridgeClient
+{
+    [RequireComponent(typeof(RosConnector))]
+    public class PosePublisher : MonoBehaviour
+    {
+        private RosSocket rosSocket;
+        public string topic = "/pose";
+        public int UpdateTime = 1;
+        private int advertizer;
+        public GameObject UnityGameObject;
```

Comment:
You dont need a GameObject below. The PoseTransformManager is what you need.
So please delete line 29 and 43 and "UnityGameobject." in line 44.